osf/models/conference.py

```diff
@@ -73,6 +79,26 @@ def __repr__(self):
     def get_by_endpoint(cls, endpoint, active):
         return cls.objects.get_by_endpoint(endpoint, active)
 
+    @property
+    def absolute_url(self):
+        return urlparse.urljoin(settings.DOMAIN, '/view/{}'.format(self.endpoint))
+
+    @property
+    def submissions(self):
+        """
+        Returns the number of valid conference submissions with at least one file attached
```

Comment:
Is this returning a number or a queryset? If it's meant to be returning a queryset, would it make more sense to just call thise from https://github.com/CenterForOpenScience/osf.io/pull/9025/files#diff-c8a3c92f6cc142037cfddb5df973a13eR119 ?

Reply:
My docstring is incorrect, I meant it to be returning a queryset! And yes, thanks for the reminder, I added this last night and forgot that I already had a similar query elsewhere -